### DISCUSSION TITLE
Can't import `tests.helpers.boring_model.BoringModelBoringModel` in colab

### DISCUSSION THREAD

**Discussion by daniellepintz** [Lightning-AI/pytorch-lightning#11681] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I am trying to import the BoringModel in this colab https://colab.research.google.com/drive/1wrPzif6zddJvdDgMnYsa05172Nv3WzWk?usp=sharing but am getting the error `ModuleNotFoundError: No module named 'tests.helpers'`

Any ideas how to fix this?

**Comment by rohitgr7** [comment, marked as answer]
with pip install it installs just the core-package (`pytorch_lightning`) and not any other directories. Since tests in this env will be just another package, not specific to pytorch_lightning.

**Reply by daniellepintz** [reply]
I see, thanks! Is there any way to import the BoringModel then?

**Reply by rohitgr7** [reply]
maybe you can try cloning the repo instead and add both paths to sys and import it directly from tests.
haven't tried on colab, but this is how I work locally with master

**Reply by daniellepintz** [reply]
Ok, thanks! I will try that

**Reply by akihironitta** [reply]
@daniellepintz Just checking if the notebook you've shared here isn't one of our official notebooks. If it is, it needs fixing :)

**Reply by daniellepintz** [reply]
@akihironitta sorry for the late reply! no, it is not one of the official notebooks
